pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -546,7 +554,10 @@ def newpkgcb(pkgId, name, arch):
             while pending:
                 done, pending = await asyncio.wait(pending, return_when=asyncio.FIRST_COMPLETED)
                 for downloader in done:
-                    results = downloader.result()
+                    try:
+                        results = downloader.result()
+                    except ClientResponseError as exc:
+                        raise HTTPNotFound(reason=_(f"File not found: {exc.request_info.url}"))
```

Comment:
You can't use f strings and gettext as the string will be interpreted before it gets translated.